CMakeLists.txt

```diff
@@ -499,6 +539,13 @@ foreach (THRUST_TEST_SOURCE IN LISTS THRUST_TESTS)
     thrust_testframework
     ${THRUST_ADDITIONAL_LIBRARIES})
 
+  if ("Feta" STREQUAL "${CMAKE_CUDA_COMPILER_ID}")
+    set_target_properties(${THRUST_TEST}
+      PROPERTIES CUDA_RESOLVE_DEVICE_SYMBOLS OFF)
+    set_target_properties(${THRUST_TEST_RDC}
+      PROPERTIES COMPILE_FLAGS "-ta:nordc")
```

Comment:
I think `-ta:nordc` should be `-gpu=nordc`

Reply:
Fixed.